apps/judicial-system/backend/src/app/modules/subpoena/guards/subpoenaExists.guard.ts

```
@@ -20,8 +21,44 @@
+
+    request.subpoena = subpoena
 
     return true
   }
 }
+
+@Injectable()
+export class SubpoenaExistsOptionalGuard extends SubpoenaExistsGuard {
+  constructor(subpoenaService: SubpoenaService) {
+    super(subpoenaService)
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)